man/chpldoc.rst

```diff
@@ -126,7 +126,6 @@ effect as passing that option once.
 BUGS
 ----
 
-See $CHPL\_HOME/STATUS for a list of known bugs and
 $CHPL\_HOME/doc/bugs.rst for instructions on reporting bugs.
```

Comment:
"See" $CHPL_HOME/doc/bugs.rst for instructions on reporting bugs.
?
(same comment as chpl.rst file)